rocky/reports/report_types/name_server_report/report.py

```diff
@@ -0,0 +1,101 @@
+from dataclasses import dataclass
+from datetime import datetime
+from logging import getLogger
+from typing import Any, Dict, List
+
+from django.utils.translation import gettext_lazy as _
+
+from octopoes.models import Reference
+from octopoes.models.ooi.dns.zone import Hostname
+from octopoes.models.ooi.network import IPAddressV4, IPAddressV6
+from reports.report_types.definitions import Report
+
+logger = getLogger(__name__)
+
+
+@dataclass
+class NameServerCheck:
+    no_uncommon_ports: bool = False
+    has_dnssec: bool = False
+    has_valid_dnssec: bool = False
+
+
+@dataclass
+class NameServerChecks:
+    checks: List[NameServerCheck]
```

Comment:
You might want to use a default factory here, such that you don't have to pass empty list literals